### PR TITLE
Ewwollesen issue3857

### PR DESCRIPTION
**Pull request by ewwollesen**
#### Summary
Added the line `KillMode=mixed` to the systemd service file in the applicable install files based on the issue linked below. Additionally, I fixed one of the files where the file format got messed up somehow.

#### Ticket Link
https://github.com/mattermost/docs/issues/3857

### CODE REVIEW

File: source/install/install-ubuntu-1804-mattermost.rst [hunk 1]
```diff
@@ -8,84 +8,144 @@ Assume that the IP address of this server is 10.10.10.2.
...
+    
     a. Change to the ``mattermost`` directory:
+
       ``cd /opt/mattermost``
+
     b. Start the Mattermost server as the user mattermost:
+
       ``sudo -u mattermost ./bin/mattermost``
+
   When the server starts, it shows some log information and the text ``Server is listening on :8065``. You can stop the server by pressing CTRL+C in the terminal window.
```
Comment (justinegeffen) [inline source/install/install-ubuntu-1804-mattermost.rst:71]:
```suggestion
  When the server starts, it shows some log information and the text ``Server is listening on :8065``. You can stop the server by pressing CTRL+C while in the terminal window.
```

**Comment by justinegeffen** [comment]
Thanks for this, @ewwollesen! Very helpful. I've added Stu, who requested this change, as a reviewer as well as our Support lead just for extra eyes. Editorially this looks good to me!

**Comment by sadohert** [comment]
Thanks for raising this @ewwollesen !

I can't seem to use the reviewer UI in Github right now.  I'm not seeing the approval part of the UI.  @justinegeffen  can you nudge my approval forward manually?

**Review by cwarnermm — APPROVED** [review]
Thank you so much for your documentation contribution. This is a great update!

**Review by justinegeffen — APPROVED** [review]

**Review by svelle — APPROVED** [review]
LGTM! Thanks for contributing Wayne! :)

**Comment by cwarnermm** [comment]
@sadohert - Thanks for reviewing this Docathon 2021 contribution for technical accuracy and completeness! Very much appreciated! Though GitHub's not letting you formally enter your approval, we have your feedback recorded.

**Pull request merged**